### PR TITLE
fix: parse dml with ```returning``` as a query

### PR DESCRIPTION
**Pull request by Eurekaaw**
Resolve issue #140
Fix a typo

Local test has passed:
image

### CODE REVIEW

File: sqllogictest-bin/src/engines/postgres.rs [hunk 1]
```diff
@@ -59,6 +59,7 @@ impl sqllogictest::AsyncDB for Postgres {
                 || lower_sql.starts_with("show")
                 || lower_sql.starts_with("with")
                 || lower_sql.starts_with("describe")
+                || lower_sql.contains("returning")
```
Comment (wangrunji0408) [inline sqllogictest-bin/src/engines/postgres.rs:62]: Constraint the command type according to https://www.postgresql.org/docs/current/dml-returning.html?
```suggestion
                || (lower_sql.starts_with("insert") || lower_sql.starts_with("update") || lower_sql.starts_with("delete")) && lower_sql.contains("returning")
```

**Comment by skyzh** [comment]
I don’t feel we are doing this correctly. Consider using https://docs.rs/postgres/latest/postgres/struct.Client.html#method.query_raw instead of matching how many rows will be returned by finding keywords in the query statement.

**Comment by skyzh** [comment]
Well, given we are using simple protocol here, probably the logic should be changed to:

* If there is at least 1 row in the result set, return DBOutput::Row.
* Otherwise, return CommandCompleted.

Which I guess could avoid matching the keywords.

**Comment by ZENOTME** [comment]
>I don’t feel we are doing this correctly. Consider using https://docs.rs/postgres/latest/postgres/struct.Client.html#method.query_raw instead of matching how many rows will be returned by finding keywords in the query statement. (For extended protocol)

Seems #142 can be fix together in this PR.

**Review by skyzh — APPROVED** [review]
We can fix Postgres extended protocol in the future. LGTM.

**Pull request merged**